docker/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM node:10.13.0-alpine as production-build
+COPY .  .
+
+# Install yarn
+RUN npm install -g yarn
+
+# Install only production dependencies
+RUN yarn install
+
+# create the production build (./build)
+RUN yarn build
+
+FROM nginx:1.13.9-alpine
+LABEL app="sfn-client"
+
```

Comment:
Very well-optimized from what I understand. 22 megabytes is a great image size compared to 600+